core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -26,6 +29,7 @@
     private final ScrollOffsetCalculator scrollOffsetCalculator;
     private final FastForwarder fastForwarder;
 
+    private Set<ScrollingPageChangeListener> onScrollingPageChangeListeners;
```

Comment:
Can we make this a List then? If Set has no advantages over List, it's a little clearer to read if we use List 😇